README.md

```diff
@@ -87,16 +87,16 @@ Again, see below for automated installation.
   `openssl req -x509 -newkey rsa:2048 -keyout key.pem -out cert.pem -days 365`
     - This will allow you to start the server but it will generate unsafe warnings in the browser. You will need a real trusted certificate for production use.
     - Adding the `-nodes` flag will generate an unencrypted private key, allowing you to run tests without passphrase prompt
-1. Install all python dependencies for hate-speech-api using `pip install -r requirements.txt` in the `hate-speech-api` directory. Start python hate-speech server using `python2 hate_speech_clf_api.py`. In `config/secrets.json`, set the `detectHateSpeech` parameter to `true`. The API will run on `http://localhost:5000`. User will never directly interact with this API.
-1. Run `npm install` from the project directory.
+2.  Hate speech detection is available for Burmese language. Set up steps are listed in Semi-automated installation script. In config/secrets.json, set the detectHateSpeech parameter to true. The API will run on http://localhost:5000. User will never directly interact with this API.
+3. Run `npm install` from the project directory.
     - This installs all dependencies and concatenates the angular application.
-1. (optional) Run `npm install -g gulp mocha karma-cli protractor migrate`.
+4. (optional) Run `npm install -g gulp mocha karma-cli protractor migrate`.
     - This installs some tools globally which can then be run from the command line for testing.
     - You will most likely need Google Chrome installed on your computer for the protractor tests to run.
     - This is optional, as `npx` provides easy access to the local copies of these that are installed by `npm install`
-1. To start server in production mode, run `npm start`. Use `npm run dev` for development.
+5. To start server in production mode, run `npm start`. Use `npm run dev` for development.
     - In your terminal, a user and password were generated. You will use these credentials to log into the application. Example: `"admin" user created with password "password"`.
-1. Navigate to `https://localhost:3000` in your browser.
+6. Navigate to `https://localhost:3000` in your browser.
```

Comment:
Can you please leave these numbers all as `1` so that we don't need to edit them all when we insert new bullet points? GitHub automatically numbers them sequentially 🙌

Reply:
yea..i think my ide changed that